### ISSUE TITLE
Tags are broken

### ISSUE THREAD

**Issue by doublemarked** [loopbackio/loopback.io#41]
For example, http://loopback.io/doc/en/contrib/tag_getting_started.html

Found on the following page: http://loopback.io/doc/en/contrib/index.html

**Comment by crandmck** [comment]
Yes, thanks for reporting. Here is what they are: http://idratherbewriting.com/documentation-theme-jekyll/mydoc_tags.html

I haven't had a chance to work on tags yet, and IMHO they are not critical, but they are "nice to have" and we should try to fix them up before we go live. I'll add this to the go-live checklist.

**Comment by crandmck** [comment]
OK, I fixed the basic problem with 2d572d1185cc36a410412989f48e84f505e89baf.  For example, see http://loopback.io/doc/en/lb2/tag_getting_started.html.  This commit adds the requisite tag archive files for both LB2 and LB3, and I modified the tag configuration a bit (see details in http://loopback.io/doc/en/contrib/tags.html#tags-overview).

I started adding tags to all the pages, but was not able to finish.  I'm going to close this issue and open a separate one for that, as we can do that time permitting.
